introduction_to_repository.md

```diff
@@ -9,7 +9,12 @@ This document aims to provide a introduction to the repository and it's structur
 
 ### Monitoring toolset
 
-The `monitoring` directory contains a set of folders containing different test suites to test different capabilities of the DSS during development and production use.
+The `monitoring` directory contains a set of folders containing different test suites to test different capabilities of the DSS during development and production use. 
+
+### Running tools locally
+
+- When running tools in the monitoring toolset for local debugging outside of Docker, the [monitoring](https://github.com/interuss/dss/tree/master/monitoring) top-level folder must be added to your `PYTHONPATH`.
```

Comment:
More precisely, the `monitoring` folder must [be accessible from the user's PYTHONPATH](https://github.com/interuss/dss/blob/8c9acd0c6659bde2c1884361776f501a17f3dd0d/monitoring/tracer/Dockerfile#L13).  How about:

```markdown
When running tools in the monitoring toolset for local debugging outside of Docker, the [monitoring](https://github.com/interuss/dss/tree/master/monitoring) folder must be accessible from one of the entries in your `PYTHONPATH`.  To accomplish this, add the root repo folder to your `PYTHONPATH`.
```

Reply:
Thanks fixed now!